tests/runtime/usdt

```diff
@@ -57,7 +57,7 @@ REQUIRES ./testprogs/systemtap_sys_sdt_check
 
 NAME usdt probes - filter probes by pid and wildcard probe name
 RUN {{BPFTRACE}} -l 'usdt:*:tracetest:test*' -p {{BEFORE_PID}}
-EXPECT usdt_test:tracetest:testprobe2
+EXPECT_REGEX usdt_test:tracetest:testprobe2
```

Comment:
Wouldn't `EXPECT` be sufficient here?

Reply:
The expected results were missing the beginning of the line here. I've updated them to be full line matching